picard/ui/options/dialog.py

```diff
@@ -172,6 +175,8 @@ def switch_page(self):
         items = self.ui.pages_tree.selectedItems()
         if items:
             page = self.item_to_page[items[0]]
+            if page.NAME != 'about':
```

Comment:
I think its awkward to have the about page open if you for example used Help -> About, close this again and then open Options -> Options just to find it to show the about page instead of the last open "real" options page.

IMHO we should probably even move the about pages to a separate dialog instead of having it in the Options dialog.

Reply:
> IMHO we should probably even move the about pages to a separate dialog instead of having it in the Options dialog.

Yes.